### DISCUSSION TITLE
Looking up formula [Sep 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3744] [Discord Threads]
> 🙋‍♂️ **Nabil Djellit**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### is possible already  
#  
> 🙋‍♂️ **Nabil Djellit**  
#### Thank you  
#  
> 🙋‍♂️ **Nabil Djellit**  
#### How can I do  
#  
> 🙋‍♂️ **Nabil Djellit**  
#### ?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1016433446656147496)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
